block/manager_test.go

```diff
@@ -93,7 +93,7 @@ func TestInitialState(t *testing.T) {
 
 func getMockDALC(logger log.Logger) da.DataAvailabilityLayerClient {
 	dalc := &mockda.DataAvailabilityLayerClient{}
-	_ = dalc.Init([8]byte{}, nil, nil, logger)
+	_ = dalc.Init([28]byte{}, nil, nil, logger)
```

Comment:
is it a dependency loop problem that we can't import this length constant from celestia?

Reply:
Restored and kept `[8]byte` for compat.